dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletionWithData/ChatWithDataRequest.cs

```diff
@@ -45,7 +45,7 @@ internal sealed class ChatWithDataRequest
 internal sealed class ChatWithDataSource
 {
     [JsonPropertyName("type")]
-    public string Type { get; set; } = ChatWithDataSourceType.AzureAISearch.ToString();
+    public string Type { get; set; } = ChatWithDataSourceType.AzureCognitiveSearch.ToString();
```

Comment:
Does this need to be an enum? Can this just be "AzureCognitiveSearch"?

Reply:
It can, I initially did that 😊.
Will drop the enum, is just one item.